src/lib/gundb/UserStorageClass.js

```diff
@@ -729,6 +729,8 @@ export class UserStorage {
     const userProperties = await this.userProperties.getAll()
     const firstVisitAppDate = userProperties.firstVisitApp
 
+    this.addBackupCard()
```

Comment:
Adding a card should not be immediately with the system cards, but after the user switches to another screen and then returns to the dashboard

Reply:
the logic is handled inside addBackupCard. there's a new 24hours logic